mmcv/onnx/symbolic.py

```diff
@@ -480,30 +479,41 @@ def register_extra_symbolics(opset=11):
     msg += reset_style
     warnings.warn(msg)
 
-    register_op('one_hot', one_hot, '', opset)
-    register_op('im2col', im2col, '', opset)
-    register_op('topk', topk, '', opset)
-    register_op('softmax', softmax, '', opset)
-    register_op('constant_pad_nd', constant_pad_nd, '', opset)
-    register_op('reflection_pad1d', reflection_pad1d, '', opset)
-    register_op('reflection_pad2d', reflection_pad2d, '', opset)
-    register_op('reflection_pad3d', reflection_pad3d, '', opset)
-    register_op('avg_pool1d', avg_pool1d, '', opset)
-    register_op('avg_pool2d', avg_pool2d, '', opset)
-    register_op('avg_pool3d', avg_pool3d, '', opset)
-    register_op('adaptive_avg_pool1d', adaptive_avg_pool1d, '', opset)
-    register_op('adaptive_avg_pool2d', adaptive_avg_pool2d, '', opset)
-    register_op('adaptive_avg_pool3d', adaptive_avg_pool3d, '', opset)
-    register_op('masked_select', masked_select, '', opset)
-    register_op('upsample_nearest1d', upsample_nearest1d, '', opset)
-    register_op('upsample_nearest2d', upsample_nearest2d, '', opset)
-    register_op('upsample_nearest3d', upsample_nearest3d, '', opset)
-    register_op('upsample_linear1d', upsample_linear1d, '', opset)
-    register_op('upsample_bilinear2d', upsample_bilinear2d, '', opset)
-    register_op('upsample_trilinear3d', upsample_trilinear3d, '', opset)
-    register_op('upsample_bicubic2d', upsample_bicubic2d, '', opset)
-    register_op('new_full', new_full, '', opset)
-    register_op('grid_sampler', grid_sampler, '', opset)
-    register_op('cummax', cummax, '', opset)
-    register_op('cummin', cummin, '', opset)
-    register_op('roll', roll, '', opset)
+    from torch.onnx import register_custom_op_symbolic
```

Comment:
Does this method require the version of pytorch?

Reply:
`register_custom_op_symbolic` works on all torch versions that we supported.